app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -71,9 +76,11 @@ import com.wire.android.ui.home.conversations.model.messagetypes.image.ImageMess
 import com.wire.android.ui.theme.wireColorScheme
 import com.wire.android.ui.theme.wireTypography
 import com.wire.android.util.CustomTabsHelper
+import com.wire.kalium.logic.data.message.Message
 import com.wire.kalium.logic.data.user.UserId
 
 // TODO: a definite candidate for a refactor and cleanup
+@Suppress("ComplexMethod")
```

Comment:
Instead of suppressing the complexity, can't it be split into smaller composable functions?

It's always a pain to read diffs of big composable functions. Especially when a new layout is introduced and the whole indentation changes.

Perhaps the whole timer thing could be extracted to its own composable instead of living in the `MessageItem` directly.